lib/maplibre-compose-material3/src/commonMain/composeResources/values-es/strings.xml

```diff
@@ -1,3 +0,0 @@
-<resources>
-  <string name="feet_symbol">pies</string>
-</resources>
```

Comment:
Was this removed due to the length of the symbol?

Reply:
sounds good to me. I'll follow up with some spanish speaking folks in the US to confirm but it's probably reasonable to remove.